### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: components/LandingPageExpo.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    let customColor = this.state.userColor
+    if (this.props.color !== undefined) {
+      customColor = this.props.color
+    }
+    const colorStyle = {
+      color: customColor
+    }
+    const { video, headline, title, des } = this.props
+    const dimensionStyle = {
+    height : Dimensions.get('window').width * .5625,
```
Comment (AdamLiechty) [inline components/LandingPageExpo.js]: Let's prefer flex layout

File: components/LandingPageExpo.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import ReactNative, {Button, NativeEventEmitter, Platform} from 'react-native';
+import client, { Color } from '@doubledutch/rn-client'
+const { TouchableOpacity, TouchableHighlight, Text, View, Image, WebView, Dimensions, Linking } = ReactNative
+
+export default class LandingPageExpo extends Component {
```
Comment (AdamLiechty) [inline components/LandingPageExpo.js]: Is this component only intended for demoing in Expo?

File: components/LandingPageExpo.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <Text style={[s.headlineText, colorStyle]}>{headline}</Text>
+        </View>
+        <View style={dimensionStyle}>
+          {/* {this.renderPlayer(video)} */}
+        </View>
+        <View style={s.box}>
+          <Text style={{textAlign:'center',fontSize:25}}>Welcome To</Text>
+          <Text style={{textAlign:'center',fontSize:25}}>{title}</Text>
+          <Text style={{textAlign:'center',fontSize:16,padding:20}}>{des}</Text>
+          <TouchableOpacity onPress={()=>{
```
Comment (AdamLiechty) [inline components/LandingPageExpo.js]: Let's fill this in or remove

File: components/SpeakerCarousel.js [hunk 4]
```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react'
 import ReactNative, { TouchableOpacity, Text, View, Image, Dimensions, Linking } from 'react-native'
-import client, {Color} from '@doubledutch/rn-client'
+import client, {Color, Avatar} from '@doubledutch/rn-client'
```
Comment (AdamLiechty) [inline components/SpeakerCarousel.js]: Avatar, used below, should set client to `client`, not `item`.

File: components/LandingPageExpo.js (hunk 1 above)
Comment (edwardstamps) [reply components/LandingPageExpo.js]: What would be a good way to maintain the height width ratio via flex?

File: components/SpeakerCarousel.js (hunk 4 above)
Comment (edwardstamps) [reply components/SpeakerCarousel.js]: Do you mean to drop using Avatar with a 3rd party image?

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (AdamLiechty) [inline .vscode/settings.json]: Let's remove this file... should put `.vscode` in `.gitignore` to keep this from being added accidentally

File: components/HomeView.js [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+export default class HomeView extends Component {
+  constructor() {
+    super()
+  }
+
+  render() {
+    const dataInput = this.props.dataInput
+    return (
+      <View title="" style={{ flex: 1,backgroundColor:'#E8E8E8' }}>
```
Comment (AdamLiechty) [inline components/HomeView.js]: remove `title=""`?

File: components/HomeView.js [hunk 7]
```diff
@@ -0,0 +1,88 @@
+"use strict";
+import React, { Component } from 'react';
+import ReactNative, { Button, ScrollView } from 'react-native';
+import client, { TitleBar, Avatar, Color } from '@doubledutch/rn-client'
+import FirebaseConnector from '@doubledutch/firebase-connector'
```
Comment (AdamLiechty) [inline components/HomeView.js]: let's remove this, and remove from package.json

File: components/HomeView.js [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+import { SmallView, SpeakerCarousel, Footer, Header, LandingPage, ButtonFooter, ImageSquares, ImageCarousel, TwoImage, OneImage, Squares, TextView, LandingPageExpo} from './index'
+const { View } = ReactNative
+
+export default class HomeView extends Component {
+  constructor() {
```
Comment (AdamLiechty) [inline components/HomeView.js]: remove empty constructor

**Pull request merged**